TodoParser.py

```diff
@@ -296,80 +374,72 @@ def _get_file_details(self, file):
                     return syntax_details, ace_mode
         return None, None
 
-    def _extract_issue_if_exists(self, comment, marker, code_block):
+    def _extract_issue_if_exists(self, comment_block, marker, hunk_info):
         """Check this comment for TODOs, and if found, build an Issue object."""
         curr_issue = None
         found_issues = []
         line_statuses = []
         prev_line_title = False
-        for match in comment:
-            comment_lines = match.group().split('\n')
-            for line in comment_lines:
-                line_status, committed_line = self._get_line_status(line)
-                line_statuses.append(line_status)
-                cleaned_line = self._clean_line(committed_line, marker)
-                line_title, ref, identifier = self._get_title(cleaned_line)
-                if line_title:
-                    if prev_line_title and line_status == line_statuses[-2]:
-                        # This means that there is a separate one-line TODO directly above this one.
-                        # We need to store the previous one.
-                        curr_issue.status = line_status
-                        found_issues.append(curr_issue)
-                    curr_issue = Issue(
-                        title=line_title,
-                        labels=[],
-                        assignees=[],
-                        milestone=None,
-                        body=[],
-                        hunk=code_block['hunk'],
-                        file_name=code_block['file'],
-                        start_line=code_block['start_line'],
-                        num_lines=1,
-                        markdown_language=code_block['markdown_language'],
-                        status=line_status,
-                        identifier=identifier,
-                        ref=ref,
-                        issue_url=None,
-                        issue_number=None
-                    )
-                    prev_line_title = True
-
-                    # Calculate the file line number that this issue references.
-                    hunk_lines = re.finditer(self.LINE_PATTERN, code_block['hunk'], re.MULTILINE)
-                    start_line = code_block['start_line']
-                    for i, hunk_line in enumerate(hunk_lines):
-                        if hunk_line.group(0) == line:
-                            curr_issue.start_line = start_line
-                            break
-                        if i != 0 and (hunk_line.group(0).startswith('+') or not hunk_line.group(0).startswith('-')):
-                            start_line += 1
-
```

Comment:
The above block was previously used to adjust the `start_line` but that is now handled in a different manner as shown above.